src/types.hpp

```diff
@@ -17,4 +17,4 @@ using f32 = float;
 using f64 = double;
 
 using Byte = u8;
-using Buffer = std::vector<Byte>;
+#define Buffer std::vector<Byte>
```

Comment:
why? using the preprocessor when you don't need to is bad

Reply:
once again for testing purposes as mentioned on discord